src/helpers/utilities.ts

```diff
@@ -45,7 +45,13 @@ function getHostInfo(): {
   // when queried from within an add-in.
   // If the platform already exposes that info, then just return it
   // (but only after massaging it to fit the return types expected by this function)
-  const context: IContext = ((window as any).Office && (window as any).Office.context) || useHostInfoFallbackLogic();
+  const isHostExposedNatively =
```

Comment:
Why do you repeatedly do the casting? It appears this wants to check if window.Office.context.host is defined, and call useHostInfoFallbackLogic() if not. Is this fix here that it checks for the Office.context.host property and not just whether Office.context is defined?

Reply:
Just pushed a change that addresses this.